style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  box-sizing: border-box;
+}
+
+html { font-size: 10px; }
+
+body {
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 2rem;
+  line-height: 1;
+  text-align: center;
+  color: black;
+  margin: 0;
+  background-color: chartreuse;
+}
+
+h1 {
+  font-size: 4rem;
+  margin-bottom: 0;
+}
+
+#main {
+  margin: 30px 8px;
+  padding: 15px;
+  border-radius: 5px;
```

Comment:
Great job! You are working on classes and id. I only want to remember you that we should use responsive values in our CSS files. These values may be like percentages instead of pixels. Example:
`margin: 15% 2%;`